pkg/server/server.go

```diff
@@ -95,6 +95,18 @@ var (
 		NativeHistogramMaxBucketNumber:  100,
 		NativeHistogramMinResetDuration: time.Hour,
 	}, []string{"grpc_service", "grpc_method", "datastore_query_count", "dispatch_count", "consistency"})
+
+	throttleSucceededCounter = promauto.NewCounter(prometheus.CounterOpts{
+		Namespace: build.ProjectName,
+		Name:      "throttled_requests_success_count",
+		Help:      "The total number of requests that have been throttled and have returned a successful response.",
+	})
+
+	throttleTimeOutCounter = promauto.NewCounter(prometheus.CounterOpts{
+		Namespace: build.ProjectName,
+		Name:      "throttled_requests_timed_out_count",
+		Help:      "The total number of requests that have been throttled but have timed-out and returned an unsuccessful responses.",
```

Comment:
should we just call it timeout rather than unsuccessful response

Reply:
We need the distinction because LU and LO will time-out and still respond with a successful status code.